services/worker/src/worker/executor.py

```diff
@@ -24,13 +25,20 @@
 
 
 async def every(
-    func: Callable[..., Optional[Any]], *args: Any, seconds: int, stop_on: Optional[Any] = None, **kwargs: Any
+    func: Callable[..., Optional[Any]],
+    *args: Any,
+    seconds: Union[int, Tuple[float, float]],
```

Comment:
```suggestion
    seconds: Union[float, Tuple[float, float]],
```

Reply:
it's what I had, but we are passing `int` to the function. I'll change this.